Snakefile.genome

```diff
@@ -167,7 +173,7 @@ rule refine:
         coalescent = "const",
         date_inference = "marginal",
         clock_rate = clock_rate,
-        root_strain = "A/skunk/NewMexico/24006483001/2024"
+        root_strain = "A/skunk/NewMexico/24-006483-001/2024"
```

Comment:
is it worth a comment here about why this strain for the root? feels kinda magic-number-y

Reply:
It's the closest outgroup.

Reply:
Added comment in https://github.com/nextstrain/avian-flu/pull/40/commits/b477716785e31c092ae75e6d9bde30ae7ee2d763